### PR TITLE
ticket 5

### PR DESCRIPTION
**Pull request by iy2k22**
I've made an attempt at Ticket 5.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ describe("GET non-existent endpoint", () => {
     return request(app)
       .get("/api/test")
       .then((res) => {
-        expect(res.body).toEqual({ msg: "error: invalid path" });
+        expect(res.body).toEqual({ msg: "error: endpoint not found" });
```
Comment (roisinoreilly) [inline __tests__/app.test.js:14]: minor point but you could use .toBe here

File: __tests__/app.test.js [hunk 2]
```diff
@@ -113,3 +113,51 @@ describe("GET /api/reviews", () => {
...
+    .expect(200);
+  })
+  test("responds with an object", () => {
+    return request(app)
+    .get("/api/reviews/1")
+    .then(({ body }) => {
+      expect(typeof body.review).toBe("object");
+    })
+  })
+  test("responds with object with appropriate properties", () => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js:123]: this is a good test but renders the previous test redundant

File: __tests__/app.test.js [hunk 3]
```diff
@@ -113,3 +113,51 @@ describe("GET /api/reviews", () => {
...
+        owner: expect.any(String),
+        review_img_url: expect.any(String),
+        review_body: expect.any(String),
+        category: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number)
+      })
+    })
+  })
+  test("responds with 404 upon being a number out of range", () => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: I'd like the test description to be more specific eg. responds with a 404 if given a valid but non-existent id

**Review by roisinoreilly — APPROVED** [review]
good work, there's a couple of small points but once they're solved merge away

**Review by robert112233 — APPROVED** [review]
Good work 👍

**Pull request merged**